DependencyInjection/Configuration.php

```diff
@@ -73,6 +73,106 @@ public function getConfigTreeBuilder()
             ->end()
         ;
 
+        $this->addResourceSection($rootNode);
+
         return $treeBuilder;
     }
+
+    /**
+     * addResourceSection
+     *
+     * @param Node $rootNode
+     * @access private
+     * @return void
+     */
+    private function addResourceSection($rootNode)
+    {
+        $rootNode
+            ->fixXmlConfig('resource')
+            ->children()
+                ->arrayNode('resources')
+                    ->prototype('array')
+                        ->children()
+                            ->scalarNode('resource_class')
+                                ->cannotBeEmpty()
+                                ->defaultValue('Dunglas\ApiBundle\Api\Resource')
+                                ->info('The resource class name')
+                            ->end()
+                            ->scalarNode('entry_class')->cannotBeEmpty()->isRequired()->info('The entity class name')->end()
```

Comment:
I am not a huge fan of the key being important to the configuration. I think it can lead to misunderstandings. If you know the bundle, you can take this shortcut, but if you are new to it, I think it is better to have an `entry_class` key.